Tasks/AppCenterDistributeV3/appcenterdistribute.ts

```diff
@@ -317,27 +319,20 @@ function beginSymbolUpload(apiServer: string, apiVersion: string, appSlug: strin
     return defer.promise;
 }
 
-function uploadSymbols(uploadUrl: string, file: string, userAgent: string): Q.Promise<void> {
+async function uploadSymbols(uploadUrl: string, file: string): Promise<void> {
     tl.debug("-- Uploading symbols...");
-    let defer = Q.defer<void>();
     tl.debug(`---- url: ${uploadUrl}`);
 
-    let stat = fs.statSync(file);
-    let headers = {
-        "x-ms-blob-type": "BlockBlob",
-        "Content-Length": stat.size,
-        "User-Agent": userAgent,
-        "internal-request-source": "VSTS"
```

Comment:
Looks like we will lose some of the headers like `User-Agent` and `internal-request-source`. But if I'm understanding correctly, this has always been the blob url, so these haven't been used before

Reply:
@DennisPan Yeah. Given that we're uploading directly to Azure before they probably weren't used. Is there anything special about these headers that we should know about?

Reply:
They were used for telemetry purpose. But looks like this particular call wasn't needed given it doesn't go through App Center